Dockerfile

```diff
@@ -1,3 +1,5 @@
+ARG node_version=18.16-slim
```

Comment:
@will-byrne this is working for me. We can revisit when we progress the deployment of this microservice.